src/generate-note-controller.js

```diff
@@ -227,15 +228,19 @@ module.exports = function(scribe){
     //validateNotes makes sure all note--start note--end and data attributes are in place
     validateNotes() {
       _.throttle(()=> {
-        mutateScribe(scribe, (focus)=> {
-          stripZeroWidthSpaces(focus);
-          config.get('selectors').forEach((selector)=>{
-            ensureNoteIntegrity(focus, selector.tagName);
-          })
-        });
+        this.ensureNoteIntegrity();
       }, 1000)();
     }
 
+    ensureNoteIntegrity(){
```

Comment:
I'm with JP. Encapsulation is one of the points of [OFO](http://harmful.cat-v.org/software/OO_programming/).